<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TournamentWindow</class>
 <widget class="QMainWindow" name="TournamentWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Football Tournament Manager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="lblTitle">
      <property name="text">
       <string>Football Tournament Manager</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="styleSheet">
       <string>QLabel { font-size: 18pt; font-weight: bold; padding: 10px; }</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBoxMatch">
      <property name="title">
       <string>Current Match</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="lblTeam1">
           <property name="text">
            <string>Team 1:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="txtTeam1">
           <property name="placeholderText">
            <string>Enter team name</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblVs">
           <property name="text">
            <string>VS</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblTeam2">
           <property name="text">
            <string>Team 2:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="txtTeam2">
           <property name="placeholderText">
            <string>Enter team name</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="lblMatchDuration">
           <property name="text">
            <string>Match Duration (minutes):</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinMatchDuration">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>120</number>
           </property>
           <property name="value">
            <number>90</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btnStartMatch">
           <property name="text">
            <string>Start Match</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnEndMatch">
           <property name="text">
            <string>End Match</string>
           </property>
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lblMatchStatus">
      <property name="text">
       <string>No match in progress</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="styleSheet">
       <string>QLabel { background-color: #e8f4f8; padding: 15px; border: 2px solid #3498db; font-size: 12pt; }</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="txtMatchLog">
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string>Match events will be logged here...</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lblNotification">
      <property name="text">
       <string>Notifications will appear here</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string>QLabel { background-color: #333333; color: white; padding: 10px; border: 1px solid #666666; }</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuLanguage">
    <property name="title">
     <string>Language</string>
    </property>
    <addaction name="actionEnglish"/>
    <addaction name="actionSpanish"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuLanguage"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionEnglish">
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="actionSpanish">
   <property name="text">
    <string>Espa√±ol</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
